### ISSUE TITLE
Hello!

### ISSUE THREAD

**Issue by avrahamm** [shincoder/homestead-docker#51]

**Comment by avrahamm** [comment]
Hello!
I followed README instructions on Windows 10,
http://localhost:8000 works fine.
Yet virtual host http://myapp.dev fails.
I created nginx virtual host in container
in /etc/nginx/sites-available/myapp.dev 

server_name myapp.dev;
    root /apps/public;

Also, in C:\\Windows\System32\drivers\etc\hosts 
added
127.0.0.1       myapp.dev
and saved.
http://myapp.dev 
fails.
-------------
This site can’t be reachedmyapp.dev refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED

------------
Yet,
$ ping myapp.dev

Pinging myapp.dev [127.0.0.1] with 32 bytes of data:
Reply from 127.0.0.1: bytes=32 time<1ms TTL=128
Reply from 127.0.0.1: bytes=32 time<1ms TTL=128
Reply from 127.0.0.1: bytes=32 time<1ms TTL=128
Reply from 127.0.0.1: bytes=32 time<1ms TTL=128

Ping statistics for 127.0.0.1:
    Packets: Sent = 4, Received = 4, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 0ms, Maximum = 0ms, Average = 0ms
----------
Any idea?

Thanks!
